source/plugin/permissions.rst

```diff
@@ -153,9 +153,9 @@ Otherwise it will be inherited from any parent ``Subject``. It does not matter w
 
 If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
 default ``Subjects``. Each ``SubjectCollection`` defines its own defaults. The global and weakest default subject can be
-obtained from the ``PermissionService``. It is recommended that plugins define their own permissions to the default's
-transient ``SubjectData`` during every server start-up. This allows the server owner to overwrite the defaults defined
-by plugins according to his needs using the default's persistent ``SubjectData``.
+obtained from the ``PermissionService``. It is recommended (but not mandatory) that plugins define their own permissions
```

Comment:
Eh, the JavaDocs need to be changed then. It's made very clear that PermissionDescriptions should not be used for inheritance or permission calculation. Same with role templates - they're not meant to be used directly.

The API (maybe?) suggests that it would be acceptable to use roles as a template, by copying their SubjectData to a group or the default subjects, but that's really the limit of their use, within the bounds of the JDs.

So to clarify, roles and permission descriptions should **not** be used for default permissions, ever. The API specifically prohibits that.